app/src/main/java/com/alexP/socialnetwork/ui/screens/main/MainActivity.kt

```diff
@@ -0,0 +1,12 @@
+package com.alexP.socialnetwork.ui.screens.main
+
+import android.view.LayoutInflater
+import com.alexP.socialnetwork.databinding.ActivityMainBinding
+import com.alexP.socialnetwork.ui.screens.base.BaseActivity
+
+class MainActivity : BaseActivity<ActivityMainBinding>() {
```

Comment:
At the moment the Activity has no logic. So keep it simple and use [alternate constructor](https://developer.android.com/reference/androidx/appcompat/app/AppCompatActivity#AppCompatActivity(int)) of AppCompatActivity.